processors/base/helpers/__init__.py

```diff
@@ -329,3 +330,85 @@ def get_canonical_location_name(location):
     logger.debug('Location "%s" normalized as "%s"', cleaned_location, current_match)
 
     return current_match
+
+
+def get_canonical_organisation_name(conn, organisation):
+    """Find a canonical organisation name according to
+    a pre-built set of clusters
+
+    Args:
+        conn (dict): connection dict
+        organisation (str): the organisation to be normalized
+    """
+    CLUSTER_QUERY = "SELECT canonical " + \
+                    "FROM organisation_clusters " + \
+                    "WHERE '%s'=ANY(variations)" % organisation
```

Comment:
Instead of binding the `organisation` parameter via string interpolation, rely on SQLAlchemy for that. The query would become `SELECT canonical FROM organisation_clusters WHERE '?'=ANY(variations)` and you'd call it as `conn['warehouse'].query(CLUSTER_QUERY, organisation)` (if I remember correctly)